samcli/commands/exceptions.py

```diff
@@ -25,6 +28,42 @@ def __init__(self, message, wrapped_from=None):
         click.ClickException.__init__(self, message)
 
 
+class UnhandledException(click.ClickException):
+    """
+    Exception class to re-wrap any exception that is not a UserException.
+    Typically this means there is a bug in SAM CLI.
+    """
+
+    GH_BUG_REPORT_URL = "https://github.com/aws/aws-sam-cli/issues/new?template=Bug_report.md&title={title}"
```

Comment:
Just to be safe, could you add a comment on the issue template that this template name shouldn't be changed, otherwise I think it would break this link:
https://github.com/aws/aws-sam-cli/blob/develop/.github/ISSUE_TEMPLATE/Bug_report.md

Reply:
Added a step in the "Unit Tests" GH Action to make sure Bug_report.md exists